### PR TITLE
add bedrock depth data

### PR DESCRIPTION
**Pull request by kmdeck**
foo

Checklist:
- [X] I created a new folder `$artifact_name`
  - [X] I added a `README.md` in that that folder that
    - [X] describes the data and processing done to it
    - [X] lists the sources of the raw data
    - [X] lists the required citation, licenses
  - [X] If applicable (e.g., for Creative Commons), I added a `LICENSE` file
  - [X] I added the scripts that retrieve, process, and produce the artifact
  - [X] I added the environment used for such scripts (typically, `Project.toml`
        and `Manifest.toml`)
  - [X] I added the `OutputArtifacts.toml` file containing the information
        needed for package developers to add `$artifact_name` to their package
- [X] I uploaded the artifact folder to the Caltech cluster (in
      `/groups/esm/ClimaArtifacts/artifacts/$artifact_name`)
- [X] I added the relevant code to the `Overides.toml` on the Caltech Cluster
      (in `/groups/esm/ClimaArtifacts/artifacts/Overrides.toml`)
- X] I added a link to the main `README.md` to point to the new artifact

### CODE REVIEW

**Comment by kmdeck** [comment]
I wonder if we could then add these files to your new PR doing the stitching?

File: bedrock_depth/create_artifact.jl [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# 60 ARC SECOND DATA
+path = FILE_PATH_60ARCSEC
+url = FILE_URL_60ARCSEC
+downloader = Downloads.Downloader()
+downloader.easy_hook =
+    (easy, info) -> Downloads.Curl.setopt(
+        easy,
+        Downloads.Curl.CURLOPT_SSL_VERIFYPEER,
+        false,
+    )
```
Comment (Sbozzolo) [inline bedrock_depth/create_artifact.jl:54]: Can you check if this is needed? This piece of code was added for a specific server that had lower security, we should not needed it for every server.

Comment (kmdeck) [reply bedrock_depth/create_artifact.jl:54]: I copied this PR almost line by line from the ETOPO2022 orography data :P @akshaysridhar was that needed there?

Comment (kmdeck) [reply bedrock_depth/create_artifact.jl]: ok, it was fine without it

File: bedrock_depth/create_artifact.jl [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    Base.mv(downloaded_file, path)
+end
+Base.cp(path, joinpath(output_dir_60arcsec, basename(path)))
+create_artifact_guided(output_dir_60arcsec; artifact_name = basename(@__DIR__) * "_60arcseconds", append = true)
+
+@info "Generated bedrock depth files"
+
+
+
+
```
Comment (Sbozzolo) [inline bedrock_depth/create_artifact.jl:55]:
```suggestion
```

Comment (Sbozzolo) [reply bedrock_depth/create_artifact.jl:55]: We don't need these lines

**Review by Sbozzolo — APPROVED** [review]

**Review by akshaysridhar — APPROVED** [review]
Thanks @kmdeck

**Pull request merged**